src/main/webapp/app/overview/course-exercises/course-exercise-row.component.html

```diff
@@ -32,6 +35,29 @@ <h4>{{ exercise.title }}</h4>
                     <jhi-result [participation]="exercise.participations[0]" short="true"></jhi-result>
                 </span>
             </div>
+            <div class="col-auto d-sm-none" [ngSwitch]="exercise.participationStatus">
+                <span class="text-muted" *ngSwitchCase="'uninitialized'">{{ 'arTeMiSApp.courseOverview.exerciseList.userNotStartedExerciseShort' | translate }}</span>
+                <span class="text-muted" *ngSwitchCase="'quiz-not-started'">{{ 'arTeMiSApp.courseOverview.exerciseList.quizNotStartedShort' | translate }}</span>
+                <span class="text-muted" *ngSwitchCase="'quiz-uninitialized'">{{ 'arTeMiSApp.courseOverview.exerciseList.userNotStartedQuizShort' | translate }}</span>
+                <span class="text-muted" *ngSwitchCase="'quiz-not-participated'">{{ 'arTeMiSApp.courseOverview.exerciseList.userNotParticipatedShort' | translate }}</span>
+                <span class="text-muted" *ngSwitchCase="'quiz-active'">{{ 'arTeMiSApp.courseOverview.exerciseList.userParticipatingShort' | translate }}</span>
+                <span class="text-muted" *ngSwitchCase="'quiz-submitted'">{{ 'arTeMiSApp.courseOverview.exerciseList.userSubmittedShort' | translate }}</span>
+                <span *ngSwitchCase="'initialized'">
+                    <jhi-result [participation]="exercise.participations[0]" short="true"></jhi-result>
+                </span>
+                <span *ngSwitchCase="'inactive'">
+                    <jhi-result [participation]="exercise.participations[0]" short="true"></jhi-result>
+                </span>
+                <span *ngSwitchCase="'quiz-finished'">
+                    <jhi-result [participation]="exercise.participations[0]" short="true"></jhi-result>
+                </span>
+                <span *ngSwitchCase="'modeling-exercise'">
+                    <jhi-result [participation]="exercise.participations[0]" short="true"></jhi-result>
+                </span>
+                <span *ngSwitchCase="'text-exercise'">
+                    <jhi-result [participation]="exercise.participations[0]" short="true"></jhi-result>
+                </span>
```

Comment:
Since this last 5 cases are the same, wouldn't be a bit shorter just assign them to the `*ngSwitchDefault` case? So you do not repeat the code too much! Just a personal style note, up to you :)

Reply:
thanks. i improved it a bit